### PR TITLE
Set actual InfluxDB field names

### PR DESCRIPTION
**Pull request by titilambert**
Reponed PR from #1138

### CODE REVIEW

**Comment by titilambert** [comment]
Last message:

> @piosz Thanks ! I understand now the schema issue. What do you think about withfields option ? This could permit smooth transitions waiting breaking change. (Of course default value should be old schema)

**Comment by titilambert** [comment]
I also rework the influxdb schema for events. We can now get all events directly from Grafana.
@piosz Should I post a Grafana dashboard example ?

**Comment by titilambert** [comment]
@piosz @mwielgus I added the `withfields` options which set new schema for influxdb sink.
By default the value is `false` so there is NO breaking changes.
New Heapster users can get the new influxdb schema with something like:
`-sink=influxdb:http://localhost:8086/k8s?withfields=true`

**Comment by titilambert** [comment]
Hello !
@mwielgus Any feedback on this one ?

File: grafana/dashboards/events_withfields.json [hunk 1]
```diff
@@ -0,0 +1,399 @@
+{
```
Comment (mwielgus) [inline grafana/dashboards/events_withfields.json]: Please move these 2 files to a subdirectory and place a readme explaining what is the difference between these and the original files.

**Comment by mwielgus** [comment]
Just one comment to apply, otherwise LGTM.

**Comment by titilambert** [comment]
@mwielgus changes done !

**Comment by titilambert** [comment]
@mwielgus do you want I squash all of this ?

**Comment by mwielgus** [comment]
Yes, please.

**Comment by titilambert** [comment]
@mwielgus squashed and rebased ;)

**Comment by mwielgus** [comment]
LGTM

**Comment by mwielgus** [comment]
Thank you for the PR (and sorry that it took so long)!

**Pull request merged**